site/website/blog/2021-02-02-jet-44-is-released.md

```diff
@@ -0,0 +1,241 @@
+Today we're releasing Hazelcast Jet 4.4 and we have some exciting new
+features!
+
+## Jet SQL
+
+Hazelcast Jet 4.4 brings you the first beta version of our SQL
+interface. You can now log into Jet from the command line and issue
+queries against the data sources you specify. They can be both data at
+rest (_batch_ sources) and live feeds (_streaming_ sources).
+
+If you have Docker at hand, here's something you can try out right now!
+(For examples that don't require Docker, go to the
+[docs](/docs/sql/intro).)
+
+```text
+docker pull hazelcast/hazelcast-jet
+docker network create jet-network
+docker run --name jet --network jet-network -v "$(pwd)":/csv-dir --rm hazelcast/hazelcast-jet
+```
+
+Wait for a message like this in the output:
+
+```text
+2021-01-15 17:50:18,645 [ INFO] [main] [c.h.c.LifecycleService]:
+    [172.17.0.2]:5701 is STARTED
+```
+
+Now start another terminal window and enter the SQL shell:
+
+```text
+$ docker run --network jet-network -it --rm hazelcast/hazelcast-jet jet --targets jet sql
+Connected to Hazelcast Jet 4.4 at [172.17.0.2]:5701 (+0 more)
+Type 'help' for instructions
+sql〉
+```
+
+You are now ready to write some SQL. Try these:
+
+```sql
+sql〉 SELECT * FROM TABLE(generate_series(1,3));
++------------+
+|           v|
++------------+
+|           1|
+|           2|
+|           3|
++------------+
+3 row(s) selected
+sql〉 SELECT key, sum(key) as total FROM (
+          SELECT v/2 as key FROM TABLE(generate_series(0, 7))
+      ) GROUP BY key;
++--------------------+--------------------+
+|                 key|               total|
++--------------------+--------------------+
+|                   0|                   0|
+|                   1|                   2|
+|                   2|                   4|
+|                   3|                   6|
++--------------------+--------------------+
+4 row(s) selected
+```
+
+Here are two more examples with streaming SQL. Streaming queries never
+complete, so use `Ctrl+C` to cancel them after a while:
+
+```sql
+sql〉 SELECT * FROM TABLE(generate_stream(10));
++--------------------+
+|                   v|
++--------------------+
+|                   0|
+|                   1|
+|                   2|
+^C
+Query cancelled.
+sql〉 SELECT * FROM TABLE(generate_stream(100)) WHERE v / 10 * 10 = v;
++--------------------+
+|                   v|
++--------------------+
+|                   0|
+|                  10|
+|                  20|
+^C
+Query cancelled.
+sql〉
+```
+
+For more examples with CSV files, Kafka and IMap, go to the
+[docs](/docs/sql/intro).
+
+We're currently very focused on bringing more features to our SQL, so
+stay tuned!
+
+## File Connector
+
+The [Unified File Connector
+API](https://jet-start.sh/docs/api/sources-sinks) gives you a simple way
+to read files, unified across different storage systems. Using the same
+API you can read files from the local filesystem, HDFS and its cloud
+derivatives such as Amazon S3, Google Cloud Storage or Azure Blob
```

Comment:
I wouldn't describe S3, GCS and ABS as derivatives of HDFS, just say

`and cloud storage systems such as ...`